api/src/migrations/1540062777613-add-project-plans.ts

```
@@ -77,10 +116,29 @@ export class addProjectPlans1540062777613 implements MigrationInterface {
+
+        await queryRunner.query(`DROP TRIGGER IF EXISTS fk_project_plan_code_delete`);
+        await queryRunner.query(`DROP TRIGGER IF EXISTS fk_project_plan_code_insert`);
+        await queryRunner.query(`DROP TRIGGER IF EXISTS fk_project_plan_code_update`);
+
         await queryRunner.query(`DROP INDEX IF EXISTS "IDX_932b5479e9af5dc8b3c0053006"`);
-        await queryRunner.query(`ALTER TABLE "project" DROP COLUMN "plan_code"`);
-        await queryRunner.query(`ALTER TABLE "project" DROP COLUMN "locales_count"`);
-        await queryRunner.query(`ALTER TABLE "project" DROP COLUMN "terms_count"`);
+
+        // Check if description column exists before including it in SELECT
+        const tableInfo = await queryRunner.query(`PRAGMA table_info("project")`);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)